src/app/app-routing.module.ts

```diff
@@ -1,10 +1,19 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ACRoutes } from './common/routes';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: ACRoutes.ManageUsers,
+    loadChildren: () =>
+      import('src/app/modules/user-manage/user-manage.module').then(
+        (mod) => mod.UserManageModule
+      ),
+  },
+];
 
 @NgModule({
   imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
+  exports: [RouterModule],
```

Comment:
`Recommendation:` avoid useless changes in your commits. It makes your code harder to read.

Reply:
It's just autoformatter )